src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -233,6 +241,18 @@ public static ConfigDef newConfigDef() {
           HADOOP_HOME_DISPLAY
       );
 
+      configDef.define(
```

Comment:
I am not sure how this orderInGroup is used internally but as we are adding this property in the middle, will it cause any issues with the already materiazlied configs? Can you find out how orderInGroup is used and if its used then we should add this as the last config?

Reply:
I couldn't find anything that would break things if the order is updated, but just to be safe I moved it to the end of the group.